includes/gateway/class-omise-payment-atome.php

```diff
@@ -191,14 +198,20 @@ private function getItems($order, $currency)
 
         // Loop through ordered items
         foreach ($items as $key => $item) {
+            // item don't have price. So we have to take subtotal and divide it by quantity to get the price
+            $pricePerItem = $item['subtotal'] / $item['qty'];
+
+            // Remove product from the list if the price is 0
+            if ((float)$pricePerItem === 0.00) {
```

Comment:
@aashishgurung mathematically, the only way for `(float)$pricePerItem` to be `0.00` is for `$item['subtotal']` to be `0` 🤔  so couldn't we leave `$pricePerItem` where it was originally defined and use `$item['subtotal']` here?